build.sbt

```diff
@@ -6,7 +6,11 @@ lazy val root = project.
   aggregate(rainierBenchmark, rainierTest).
   aggregate(shadedAsm).
   settings(commonSettings).
-  settings(unpublished)
+  settings(unpublished).
+  settings(
+    // crossScalaVersions must be set to Nil on the aggregating project
+    crossScalaVersions := Nil
```

Comment:
This is following the advice of https://www.scala-sbt.org/1.x/docs/Cross-Build.html#Cross+building+a+project